scripts/create-database.js

```diff
@@ -0,0 +1,54 @@
+const { Client } = require('pg');
+const path = require('path');
+
+// capture first command line argument passed to this script
```

Comment:
This file looks to be copy pasted from the track materials. Even if you do not understand fully what is going on here, blindly copy pasting is not a good thing as you could be adding bad or even malicious code, and creating side effects by accident that you may not even be aware of at first.

Its much more beneficial to you while learning these things to get into the idea of typing your work out as it helps you get accustomed to flow of work, and also is more engaging mentally. You should remove the comments here, especially the one about moving variables to another file on line 41